app/Console/Commands/AddFirstLibrariand.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Models\User;
+use Illuminate\Console\Command;
+
+class AddFirstLibrariand extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'librarian:add';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Add the first librarian to the system';
+
+    /**
+     * Execute the console command.
+     */
+    public function handle()
+    {
+        $name = $this->ask('Enter librarian name');
+        $username = $this->ask('Enter librarian username');
+        $email = $this->ask('Enter librarian email');
+        $jmbg = $this->ask('Enter librarian jmbg');
+        $password = $this->ask('Enter librarian password min 8 and max 16 characters');
+
+        // ... ask for last name, email, username, jmbg and validate them
```

Comment:
Please add validation, for example JMBG must be 13 digits, email must not be taken and must be in email form

Reply:
Validation added for:
- jmbg
- username
- email